docs/integrations.md

```diff
@@ -0,0 +1,15 @@
+# Integrations
+
+This documentation describes major tools related to RBS.
+
+## Type Checkers
+
+* [Steep](https://github.com/soutaro/steep)
+* [TypeProf](https://github.com/ruby/typeprof)
+
+## Editor integrations
+
+* Emacs: [rbs-mode](https://github.com/ybiquitous/rbs-mode)
+* RubyMine: [It supports RBS by default](https://www.jetbrains.com/help/ruby/rbs.html)
+* Vim: [rbs.vim](https://github.com/pocke/rbs.vim)
+* Visual Studio Code: [vscode-rbs-syntax](https://github.com/soutaro/vscode-rbs-syntax)
```

Comment:
How about adding Sublime Text plugin? https://github.com/soutaro/sublime-rbs-plugin

Reply:
Ah, I didn't know (or I forgot) this plugin. I added it 👍 
d21a036a268219cf64fd5be29fea5fa1de62dd94